<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>421</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>7</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>8</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>10</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>New Column</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Role</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>430</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>All Employees</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Employees born in 2001</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Top 3 Oldest Employees</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Exclude Testers</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Count Employees by Role</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>480</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Employee Data</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
